.packit.yaml

```diff
@@ -6,40 +6,211 @@ merge_pr_in_ci: false
-    - bash -c "grep -m1 '^Version:' packaging/leapp.spec | grep -om1 '[0-9].[0-9.]**'"
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      owner: "@oamg"
+      project: leapp
+      targets:
+        - epel-7-ppc64le
+        - epel-7-x86_64
+        - epel-8-x86_64
+        - fedora-all-aarch64
+        - fedora-all-x86_64
+    actions:
+      post-upstream-clone:
+        # builds from PRs should have lower NVR than those from master branch
+        - sed -i "s/1%{?dist}/0%{?dist}/g" packaging/leapp.spec
+      get-current-version:
+        # get version from spec file instead from git tag
+        - bash -c "grep -m1 '^Version:' packaging/leapp.spec | grep -om1 '[0-9].[0-9.]**'"
+
+  - job: copr_build
+    trigger: commit
+    metadata:
+      branch: master
+      owner: "@oamg"
+      project: leapp
+      targets:
+        - epel-7-ppc64le
+        - epel-7-x86_64
+        - epel-8-x86_64
+        - fedora-all-aarch64
+        - fedora-all-x86_64
+    actions:
+      post-upstream-clone:
+        # builds from master branch should have the highest NVR
+        - sed -i "s/1%{?dist}/100%{?dist}/g" packaging/leapp.spec
+      get-current-version:
+        # get version from spec file instead from git tag
+        - bash -c "grep -m1 '^Version:' packaging/leapp.spec | grep -om1 '[0-9].[0-9.]**'"
+
+  - job: tests
+    fmf_url: "https://gitlab.cee.redhat.com/oamg/leapp-tests"
+    fmf_ref: "master"
+    use_internal_tf: True
+    trigger: pull_request
+    targets:
+      epel-7-x86_64:
+        distros: [RHEL-7.9-ZStream]
+    identifier: tests-7.9to8.6
+    tmt_plan: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*8to9)(?!.*max_sst)"
+    tf_post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys; yum install -y \"leapp-upgrade*master*\""
+    tf_extra_params:
+      environments:
+        - tmt:
+            context:
+              distro: "rhel-7.9"
+    env:
+      SOURCE_RELEASE: "7.9"
+      TARGET_RELEASE: "8.6"
+
+  - job: tests
+    fmf_url: "https://gitlab.cee.redhat.com/oamg/leapp-tests"
+    fmf_ref: "master"
+    use_internal_tf: True
+    trigger: pull_request
+    targets:
+      epel-7-x86_64:
+        distros: [RHEL-7.9-ZStream]
+    identifier: tests-7.9to8.8
+    tmt_plan: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*8to9)(?!.*max_sst)"
+    tf_post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys; yum install -y \"leapp-upgrade*master*\""
+    tf_extra_params:
+      environments:
+        - tmt:
+            context:
+              distro: "rhel-7.9"
+    env:
+      SOURCE_RELEASE: "7.9"
+      TARGET_RELEASE: "8.8"
+
+  - job: tests
+    fmf_url: "https://gitlab.cee.redhat.com/oamg/leapp-tests"
+    fmf_ref: "master"
+    use_internal_tf: True
+    trigger: pull_request
+    targets:
+      epel-7-x86_64:
+        distros: [RHEL-7.9-ZStream]
+    identifier: tests-7.9to8.8-sst
```

Comment:
sst tests take something close to eternity, it would be great if it was possible to run them on demand, not as part of every `/packit build` or `/packit test`. Now we have '/rerun-sst' that will schedule just sst tests while `/rerun` runs our usual regression tests. Do you think it will be possible to have the same behavior with packit?

Reply:
Currently, I don't think so... I haven't seens anything similar like that in their docs, but I know that I (we) suggested that packit could have more comments commands available to be used. Maybe we should open an issue for them regarding it? 

I think it could be something like: 

```yaml
-job: tests
 fmf_url: ""
 fmf_ref: ""
 use_internal_tf: True
 trigger: comment
   command: "/packit run <your-test>"
 ...
```

What do you think?

Reply:
We already have an issue for this https://github.com/packit/packit-service/issues/1806 , feel free to express interest there :) I don't think it has a high priority at the moment but if there is demand for this, I guess that may change.

Reply:
Thanks, @FrNecas!